raisinbread/README.md

```diff
@@ -0,0 +1,56 @@
+    SQL/0000-00-04-Help.sql \
+    SQL/0000-00-05-ElectrophysiologyTables.sql \
+    raisinbread/instruments/instrument_sql/aosi.sql \
+    raisinbread/instruments/instrument_sql/bmi.sql \
+    raisinbread/instruments/instrument_sql/medical_history.sql \
+    raisinbread/instruments/instrument_sql/mri_parameter_form.sql \
+    raisinbread/instruments/instrument_sql/radiology_review.sql \
+    test/test_instrument.sql \
+    raisinbread/RB_files/*.sql | mysql
+```
+
+##### Configuring
+In order to be able to load the LORIS front-end while using the RaisinBread dataset some configurations are necessary.
+
+1. Copy the `raisinbread/config.config.xml` file into `project/config.xml`
+2. Input the correct `<database>` information in the `project/config.xml` file 
+3. Change the values the the `Config` table of the SQL database to reflect the correct `host` and `base` values
+4. copy the `raisinbread/instruments/` instrument PHP and LINST files to the `projects/instruments/` directory
+
+### Modifying RB
+The RaisinBread database should be handled like any other project. The data should never be modified directly in the SQL files, the database should be sourced, altered and re-exported into SQL files.
+
+When possible, data modifications should be done directly from the LORIS front-end or using the loris tools. When modifications need to be done directly in SQL, it is important to only add clean data with proper realistic logical values where possible.
+
+When implementing a feature that requires SQL modifications, the SQL patch created and submitted to the LORIS repository should be run on RaisinBread just like it would be run on any project database. The changes should appear in the same pull request as the new feature on the LORIS repository.
+
+Any modification to the database will be reflected in the exports, make sure no personal information gets accidentally added to the RaisinBread database while you are modifying it.
+
+
+### Exporting RB
+The RaisinBread data should be updated with every code change impacting the database. To do so, the user needs to source the database before the changes, modify the database and then export the data to be submitted back to LORIS in the same pull request as the code changes.
+
+##### Contributing back into LORIS
+In order to submit changes made to RaisinBread back to LORIS, an export tool was created in the `tools/exporters/` directory. The `DB_dump_table_data.php` PHP script executes an advanced `mysqldump` command that generates a single SQL file per SQL table in the `raisinbread/RB_files/` directory. Each file contains foreign keys disabling and enabling, table truncation to clear all existing data and table locking to avoid data corruption during sourcing. If the data has been modified between sourcing these files and exporting them, the modification will be reported as git uncommitted changes and thus they can be verified and submitted to the LORIS repo in the same pull request as the code.
```

Comment:
What makes it "advanced"?

Reply:
it has a bunch of parameters, replace with "complex" ?

Reply:
I don't think any extra adjective at all is required.